sdcm/utils/loader_utils.py

```diff
@@ -106,8 +115,15 @@ def _run_all_stress_cmds(self, stress_queue, params):
             stress_params.update({'stress_cmd': stress_cmd})
 
             # Due to an issue with scylla & cassandra-stress - we need to create the counter table manually
+            # also tables doesn't yet support counters, so we skip the command and error about it
             if 'counter_' in stress_cmd:
-                self._create_counter_table()
+                if self.tablets_enabled and SkipPerIssues('scylladb/scylladb#18180', params=self.params):
+                    TestFrameworkEvent(severity=Severity.ERROR,
```

Comment:
no sure if we should error like that, or just warning about it ? 

anyhow it's better then before that it was failing the test completely, like that it would run the whole test, and fail it in the end cause of the error event

Reply:
it make sense to emit such and error and fail the test.
No sure anyone will bother looking at warnings.

Reply:
I don't know if it's better.
Do we have tests that mixes both counters and other stress types?

I know we have some dedicated for counters, will they run without stress or just finish quickly?
In any case, they should be "Red" so one will know they didn't run as planned.

Reply:
checking it again, we don't have such test cases

it appear only on the provision test, and in some other unit test configurations...

which is quite weird...

Reply:
I've found we only have 2 tests covering counters and they are using scylla-bench only for some reason.

so this fix isn't that important, it's just hack for when we use c-s (maybe it didn't create the table correctly at some point...)

Reply:
it looks like `test-cases/longevity/longevity-counters-multidc.yaml` and `test-cases/longevity/longevity-counters-3h.yaml` are not tested.

Reply:
Why does it look so?
They are tested during releases.

Reply:
So if one runs them now with tablets they will fail the same, right?

Which is probably good, because it can be used as a negative test in case someone wants it.

For 2025.1 they will need to be tested only for "tablets-disabled" variation.
@pehala FYI

Reply:
Anyhow i'm merging it, it doesn't really affect any real test case, just the one used in provisioning test